File: src/Generated20260206.go
package main

import "fmt"

type Generated20260206 struct {
    // Generated struct 20260206_062653_559627
    Value int
    Created string
}

func (c *Generated20260206) Process() int {
    result := c.Value
    for i := 0; i < 21; i++ {
        result += i * 7
    }
    return result
}

func main() {
    obj := Generated20260206{Value: 880, Created: "20260206_062653_559627"}
    fmt.Println(obj.Process())
}

File: src/Generated20260206.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260206 {
public:
    // Generated class 20260206_062755_749730
    int value = 889;
    string created = "20260206_062755_749730";
    
    int process() {
        int result = value;
        for (int i = 0; i < 26; i++) {
            result += i * 2;
        }
        return result;
    }
};

int main() {
    Generated20260206 obj;
    cout << obj.process() << endl;
    return 0;
}